src/chapters/exceptions/exercises.rst

```diff
@@ -0,0 +1,29 @@
+Exercises
+=========
+
+To get started, fork and clone this `repo <https://github.com/LaunchCodeEducation/csharp-web-dev-lsn9exceptions>`_ from Github. 
+
+Divide By Zero!
+---------------
+
+Complete a function called ``Divide()`` in ``Program.cs``.
```

Comment:
![image](https://user-images.githubusercontent.com/20828430/80612784-111e7d80-8a02-11ea-87c4-f50b303aa743.png)
the way these two sentences get rendered together makes Program.cs.Divide() look like one entity. Change the second to say "the method..."?

Reply:
Done!